providers/efa/efa.h

```diff
@@ -24,6 +24,7 @@ struct efa_context {
 	uint32_t max_llq_size;
 	size_t cqe_size;
 	struct efa_qp **qp_table;
+	int qp_table_sz_m1;
```

Comment:
unsigned type, do not use signed types in bit manipulations.

Reply:
Done